client/blocks/reader-avatar/index.tsx

```diff
@@ -25,6 +25,7 @@ type ReaderAvatarProps = {
 	preferBlavatar?: boolean;
 	showPlaceholder?: boolean;
 	isCompact?: boolean;
+	isRecentFeed?: boolean;
```

Comment:
Instead of adding an additional specific prop, what do you think about exposing props that affect these sizes directly? Something like `siteIconSize` and `gravatarSize`? It might be overkill but it could help us keep prop API bloat down if we think these sorts of changes will be needed in the future.

Reply:
Thanks for pointing this out, I was looking at `isCompact`, but the component actually already has an `iconSize` prop. 🎉 Updated in https://github.com/Automattic/wp-calypso/pull/96071/commits/68113e7555218c482278dfbc524f115057e90eb9